Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:20.04
```

Comment:
alpine

Reply:
Cant use alpine need glibC, for now lets use Ubuntu base and change it late?

Reply:
That works for me.